<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>461</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="gameLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
   </layout>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>87</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Запустить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>87</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Остановить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>87</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Очистить</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="randomFillingBox">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>149</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Случайное заполнение</string>
   </property>
   <widget class="QSpinBox" name="blackChance">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="specialValueText">
     <string/>
    </property>
    <property name="suffix">
     <string>%</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>40</number>
    </property>
   </widget>
   <widget class="QPushButton" name="randomFillButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>87</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Заполнить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Живых клеток:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="loadSaveBox">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>39</y>
     <width>271</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Загрузка/сохранение</string>
   </property>
   <widget class="QPushButton" name="loadGameButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>87</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Загрузить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveGameButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>87</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить</string>
    </property>
   </widget>
   <widget class="QComboBox" name="stateSelection">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>271</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Своя фигура</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gosper's Glider Gun</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pulsar</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Фигуры:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="settingsBox">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>200</y>
     <width>271</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Параметры</string>
   </property>
   <widget class="QCheckBox" name="wrapping">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Замкнуть поле</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>62</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Скорость:</string>
    </property>
   </widget>
   <widget class="QSlider" name="evolutionSpeed">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="pageStep">
     <number>100</number>
    </property>
    <property name="value">
     <number>30</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="fieldSize">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>30</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Размер стороны поля:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
